docs/contribute.rst

```diff
@@ -0,0 +1,105 @@
+.. include:: ../.github/CONTRIBUTING.rst
+
+
+First-time setup
+~~~~~~~~~~~~~~~~
+
+- `Install git`_
+
+- `Configure git`_:
+
+1. Please identify yourself::
+
+        $ git config --global user.name "firstname lastname"
+
+        $ git config --global user.email yourname@example.com
```

Comment:
```suggestion
        $ git config --global user.email yourname@example.com

* Use the address bound to your GitHub account so that the commits would be linked to your profile.
```